homeassistant/components/airzone_cloud/sensor.py

```diff
@@ -129,6 +167,27 @@ def __init__(
         self._async_update_attrs()
 
 
+class AirzoneWebServerSensor(AirzoneWebServerEntity, AirzoneSensor):
+    """Define an Airzone Cloud WebServer sensor."""
+
+    def __init__(
+        self,
+        coordinator: AirzoneUpdateCoordinator,
+        description: SensorEntityDescription,
+        entry: ConfigEntry,
+        ws_id: str,
+        ws_data: dict[str, Any],
+    ) -> None:
+        """Initialize."""
+        super().__init__(coordinator, entry, ws_id, ws_data)
+
+        self._attr_name = f"{ws_id} {description.name}"
+        self._attr_unique_id = f"{entry.unique_id}_{ws_id}_{description.key}"
```

Comment:
This seems odd? If the unique ID of an config entry is already used, why does the ws_id matter?

Reply:
Because the ws_id is the MAC address of the WebServer corresponding to that sensor, and since an installation can have multiple webservers, we need to be able to add more than 1, as opposed to the local integration in which there's only 1 of them.

Reply:
But the webserver is already unique by itself...

Reply:
> But the webserver is already unique by itself...
> 
> 
> 
> So either way, it seems odd to see both of them?

Ah, now I get it, so you mean that I should remove the unique_id and just use the ws_id, right?
(Sorry, I didn't get your point at first).

Reply:
BTW, in that case we could also have used aidoo_id and zone_id for AirzoneAidooSensor and AirzoneZoneSensor since they are unique too...

Reply:
Hehe we don't have to make an ID more unique than unique 😉

Reply:
> Hehe we don't have to make an ID more unique than unique 😉

Do you want me to take care of that and change AirzoneAidooEntity and AirzoneZoneEntity in this or a new PR since this hasn't been released yet to the public?

Reply:
Done for AirzoneWebServerEntity in 64f75e3c9f714148841cbf727f5ee48e14fb2485

Reply:
I think that would make sense, especially since it hasn't reached stable yet. It means we can improve without the need for migrations at this point. 👍

Reply:
> I think that would make sense, especially since it hasn't reached stable yet. It means we can improve without the need for migrations at this point. 👍

Done in 62a308231db0d23accb9c4b1556eae3451a2d20d